sdk/communication/azure-communication-jobrouter/azure/communication/jobrouter/_models.py

```diff
@@ -1311,41 +1321,11 @@ def __init__(
         :keyword suspend_mode: Any object.
         :paramtype suspend_mode: JSON
         """
-        super().__init__()
```

Comment:
This is only needed if your model is inheriting from something - so most of them wont.